integrations/Gemfile

```diff
@@ -41,7 +41,7 @@ gem "async-websocket", "~> 0.8.0"
 
 gem "git"
 
-gem "hubspot-api-client"
+gem "hubspot-api-client", "~> 17.2.0"
```

Comment:
Why do we need this change and how is this related with AWS Athena connector

Reply:
I needed to add aws-sdk-athena but if I gem install hubspot will become a version higher than 17.2.0 causing the hubspot client_rspec to fail. In short this is used to prevent the hubspot fail on my end.